### PR TITLE
RPC update part -2 : Removing account api

### PR DESCRIPTION
**Pull request by Naviabheeman**
This PR is after #150. 
all 'account' and 'dummy' parameters are removed from the RPC layer
Test cases are updated

### CODE REVIEW

**Comment by azuchi** [comment]
@Naviabheeman can you rebase this?

**Review by azuchi — CHANGES_REQUESTED** [review]
`prioritisetransaction` still has `dummy`.

File: src/wallet/rpcwallet.cpp [hunk 1]
```diff
@@ -398,99 +345,6 @@ static UniValue setlabel(const JSONRPCRequest& request)
...
-        const CTxDestination& dest = item.first;
-        const std::string& strName = item.second.name;
-        if (strName == strAccount) {
-            ret.push_back(EncodeDestination(dest));
-        }
-    }
-    return ret;
-}
-
 static CTransactionRef SendMoney(CWallet * const pwallet, const CTxDestination &address, CAmount nValue, bool fSubtractFeeFromAmount, const CCoinControl& coin_control, mapValue_t mapValue, std::string fromAccount, const ColorIdentifier& colorId)
```
Comment (azuchi) [inline src/wallet/rpcwallet.cpp:348]: The argument `fromAccount` is still here.

**Comment by Naviabheeman** [comment]
> `prioritisetransaction` still has `dummy`.

@azuchi, can we remove this RPC? it seems to be deprecated.

**Comment by azuchi** [comment]
> > `prioritisetransaction` still has `dummy`.
> 
> @azuchi, can we remove this RPC? it seems to be deprecated.

@Naviabheeman OK.

**Review by azuchi — APPROVED** [review]

**Pull request merged**